macOS/DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/VPNAppEventsHandler.swift

```diff
@@ -71,19 +71,14 @@ final class VPNAppEventsHandler {
     ///
     private func loginItemsControlCheckpoint(canRestart: Bool) {
         Task { @MainActor [loginItemsManager] in
-            guard loginItemsManager.isAnyEnabled(LoginItemsManager.vpnLoginItems) else {
-
-                return
-            }
-
             switch try? await featureGatekeeper.canStartVPN() {
-            case .some(true):
+            case .some(true) where loginItemsManager.isAnyEnabled(LoginItemsManager.vpnLoginItems):
                 if canRestart {
                     restartLoginItem(using: loginItemsManager)
                 }
-            case .some(false):
+            case .some(false) where loginItemsManager.isAnyInstalled(LoginItemsManager.vpnLoginItems):
```

Comment:
If the user can't start the VPN, but the login item is installed, we should disable the login item.